src/Concerns/ManagesDataDefinitions.php

```diff
@@ -30,28 +30,6 @@ trait ManagesDataDefinitions
      */
     abstract public function getSpannerDatabase(): Database;
 
-    /**
-     * @deprecated use runDdlBatch() instead
-     * @param string $ddl
-     * @return LongRunningOperation
-     */
-    public function runDdl(string $ddl): LongRunningOperation
-    {
-        trigger_deprecation('colopl/laravel-spanner', '5.2', 'runDdl() is deprecated. Use runDdlBatch() instead.');
```

Comment:
`use function trigger_deprecation;` is no longer used.

Reply:
deleted.